### PR TITLE
Update parent pom version

### PR DESCRIPTION
**Pull request by vinayvishal**
Updated required dependency versions.
Added nexus-staging plugin.
Updated parent pom version to 1.0.5

Signed-off-by: Vinay Vishal <vinay.vishal@oracle.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -104,6 +104,15 @@
                         </exclude>
                     </configuration>
                 </plugin> 
+                <plugin>
```
Comment (bshannon) [inline pom.xml]: I don't think this change is needed when using the 1.0.5 parent pom.

Comment (romain-grecourt) [reply pom.xml]: correct.

Comment (vinayvishal) [reply pom.xml]: Done.

File: pom.xml [hunk 2]
```diff
@@ -19,6 +19,12 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.5</version>
```
Comment (romain-grecourt) [inline pom.xml]: Use <relativePath/> to avoid the warning about the parent pom not available at ../pom.xml

Comment (vinayvishal) [reply pom.xml]: Done.

**Review by romain-grecourt — APPROVED** [review]

**Pull request merged**